firstvoices/backend/resources/join_requests.py

```diff
@@ -0,0 +1,71 @@
+from import_export import fields
+from import_export.results import RowResult
+from import_export.widgets import ForeignKeyWidget
+from jwt_auth.models import User
+
+from backend.models.join_request import (
+    JoinRequest,
+    JoinRequestReason,
+    JoinRequestReasonChoices,
+    JoinRequestStatus,
+)
+from backend.resources.base import SiteContentResource
+from backend.resources.utils.import_export_widgets import ChoicesWidget
+
+
+class JoinRequestResource(SiteContentResource):
+    user = fields.Field(
+        column_name="user",
+        attribute="user",
+        widget=ForeignKeyWidget(User, "email"),
+    )
+
+    status = fields.Field(
+        column_name="status",
+        attribute="status",
+        widget=ChoicesWidget(JoinRequestStatus.choices),
+    )
+
+    class Meta:
+        model = JoinRequest
+
+    def import_row(
```

Comment:
Join requests with unknown users are currently skipped (as you must have an account/be logged in to submit a join request). 

However this could be changed to create a user model for the email if desired.